src/cmap/connection.ts

```diff
@@ -187,21 +197,21 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
   [kClusterTime]: Document;
 
   /** @event */
-  static readonly COMMAND_STARTED = 'commandStarted' as const;
+  static readonly COMMAND_STARTED = COMMAND_STARTED;
```

Comment:
Just a question here. We are importing constants here and then redefining them as static variables on this class. Why not just use the original constants everywhere?